### PR TITLE
Skeleton for HTTP2 Specification

### PR DESCRIPTION
**Pull request by krismcqueen**
Included is the skeleton of tests for the HTTP2 Specification.  There is a single fully implemented test, which is a simple HTTP2 negotiation over plain text.  Most of the work has been filed across 9 different issues to fully implement the test suite.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+        <dependency>
+            <!-- Needed for EL only, not AST -->
+            <groupId>org.kaazing</groupId>
+            <artifactId>k3po.lang</artifactId>
+            <version>[1.99.99.99,2.1.0.0)</version>
+        </dependency>
+        <dependency>
+            <groupId>commons-codec</groupId>
+            <artifactId>commons-codec</artifactId>
+            <version>1.6</version>
```
Comment (dpwspoon) [inline pom.xml]: FYI, This is an old version I believe.  commons-codec is on 1.10

**Comment by krismcqueen** [comment]
From @dpwspoon: consider adding the Google license plugin that adds the license header if required; consider adding the NOTICE.txt in the event there is a third party license that needs to be copied as part of copying the spec project.

File: src/main/scripts/org/kaazing/specification/http2/negotiate/negotiate.http2.connection.request.rpt [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+# write the HTTP2-Settings header for the SETTINGS_HEADER_TABLE_SIZE
+#   setting set to 8192 octets (the default is 4096)
+write header "HTTP2-Settings" ${http2:settingsWithHeaderTableSize8192()}
+
+read status "101" /.+/
+read version "HTTP/1.1"
+read header "Upgrade" "h2c"
+read header "Connection" "Upgrade"
+
+# connection established
```
Comment (jfallows) [inline src/main/scripts/org/kaazing/specification/http2/negotiate/negotiate.http2.connection.request.rpt]: We've been standardizing on having a LF at the end of each script file.

File: src/main/scripts/org/kaazing/specification/http2/negotiate/negotiate.http2.connection.response.rpt [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+#     TODO: validate that it is SETTINGS_HEADER_TABLE_SIZE set to 8192 octets
+read header "HTTP2-Settings" /(?<settings>[a-zA-Z0-9\-_=]{6})/
+
+write status "101" "Switching Protocols"
+write version "HTTP/1.1"
+write header "Upgrade" "h2c"
+write header "Connection" "Upgrade"
+write flush
+
+# connection established
```
Comment (jfallows) [inline src/main/scripts/org/kaazing/specification/http2/negotiate/negotiate.http2.connection.response.rpt]: We've been standardizing on having a LF at the end of each script file.

**Pull request merged**